docker/README.md

```diff
@@ -53,6 +53,11 @@ All images are available at https://hub.docker.com/r/tinode/
 	$ docker run -p 6060:18080 -d --name tinode-srv --network tinode-net tinode/tinode-mongodb:latest
 	```
 
+	You can also run Tinode with the `tinode/tinode-all` image (which has all of the above DB adapters compiled in). You will need to specify the database adapter using the `ADAPTER_NAME` environment variable. E.g. for `mysql`, the command line will look like
```

Comment:
Maybe name it `tinode/tinode` like everyone else does.

Reply:
Done.